setup.cfg

```diff
@@ -44,8 +45,10 @@ evm_transition_tool =
 
 [options.entry_points]
 console_scripts =
-    fill = entry_points.fill:main
-    tf = entry_points.tf:main
+    fill = entry_points.cli:fill
+    phil = entry_points.cli:fill
+    tf = entry_points.cli:tf
```

Comment:
I'm up for deprecating `tf` now if you are

Reply:
Before the workshop I would have :100: agreed! But someone there literally mentioned "calling tf". So inclined to leave it for. Doesn't really hurt.